src/main/java/org/terasology/hunger/HungerAuthoritySystem.java

```diff
@@ -69,6 +80,12 @@ public void update(float delta) {
         }
     }
 
+    /**
+     * Cancels health regen for an entity if their hunger is below the threshold for healing health.
```

Comment:
not sure about the formatting, may need a newline between description and params.